### DISCUSSION TITLE
账号显示被封禁，无法使用

### DISCUSSION THREAD

**Discussion by wenji1992** [BurnHair/docs#112] [技术支持]
![360截图17001022100127134](https://github.com/BurnHair/docs/assets/163493355/c5550c8f-b0f3-473a-9dea-31c4a81b6833)
请问是被封禁了吗，账号是：wenji1990。能否麻烦解封一下。另外商城能否上架一些1、2USD的商品，感谢

**Comment by BennyThink** [comment]
封禁的原因是重复注册，可用的账号是 wenji1991 。而且现在起来，wenji1990 和 wenji1992，就很怪你不觉得吗？
